pom.xml

```diff
@@ -86,11 +85,12 @@
       <artifactId>processors</artifactId>
       <version>1.0-SNAPSHOT</version>
       <scope>test</scope>
-  </dependency>
-  <dependency>
+    </dependency>
+    <dependency>
```

Comment:
Here as well.

Reply:
There was a misalignment in the original file. If a minimised diff is needed, I can remove it.